sklearn/metrics/_classification.py

```diff
@@ -2431,9 +2432,7 @@ def brier_score_loss(y_true, y_prob, sample_weight=None, pos_label=None):
         Sample weights.
 
     pos_label : int or str, default=None
-        Label of the positive class.
-        Defaults to the greater label unless y_true is all 0 or all -1
-        in which case pos_label defaults to 1.
+        The label of the positive class.
```

Comment:
Why remove the rest of the docstring? That behaviour should still be respected

Reply:
we never support automatically inferring pos_label, so this docstring is actually wrong.

Reply:
Yes we did `pos_label = y_true.max()`.